Content.Server/GameObjects/Components/Arcade/SpaceVillainArcadeComponent.cs

```diff
@@ -306,20 +347,23 @@ private bool CheckGameConditions()
             {
                 if ((_enemyHp <= 0 || _enemyMp <= 0) && (_playerHp > 0 && _playerMp > 0))
                 {
-                    UpdateUi("You won!", $"{_enemyName} dies.");
+                    _running = false;
+                    UpdateUi(Loc.GetString("You won!"), Loc.GetString("{0} dies.", _enemyName), true);
                     EntitySystem.Get<AudioSystem>().PlayFromEntity("/Audio/Effects/Arcade/win.ogg", Owner.Owner, AudioParams.Default.WithVolume(-4f));
                     Owner.ProcessWin();
                     return false;
                 }
                 if ((_playerHp <= 0 || _playerMp <= 0) && _enemyHp > 0 && _enemyMp > 0)
                 {
-                    UpdateUi("You lost!", $"{_enemyName} cheers.");
+                    _running = false;
+                    UpdateUi(Loc.GetString("You lost!"), Loc.GetString("{0} cheers.",_enemyName), true);
                     EntitySystem.Get<AudioSystem>().PlayFromEntity("/Audio/Effects/Arcade/gameover.ogg", Owner.Owner, AudioParams.Default.WithVolume(-4f));
                     return false;
                 }
                 if ((_playerHp <= 0 || _playerMp <= 0) && (_enemyHp <= 0 || _enemyMp <= 0))
```

Comment:
You could just put those two into one `if ((_playerHp <= 0 || _playerMp <= 0)`

Reply:
i dont get it?